libs/brig-types/src/Brig/Types/Swagger.hs

```diff
@@ -150,6 +154,12 @@ user = defineModel "User" $ do
         description "Unique user handle."
         optional
 
+managedBy :: DataType
+managedBy = string $ enum
+    [ "wire"
+    , "scim"
+    ]
```

Comment:
Nice, i didn't know this when i wrote the swagger `DataType` for `Role`, will fix.

Is the following better (automatically keeps in sync with changes in the type) or worse (convoluted and hard to read)?  If better, I will change this everywhere in the code-base in a separate PR.

```
dataTypeFromBoundedEnum :: forall a proxy. (Bounded a, Enum a, Aeson.ToJSON a) => proxy a -> DataType
dataTypeFromBoundedEnum _ = string . enum $ unpack . Aeson.encode <$> [(minBound :: a)..]

role :: DataType
role = dataTypeFromBoundedEnum (Proxy @Role)
```

Reply:
I'd say it's better as long as `dataTypeFromBoundedEnum` has a haddock explaining that the Aeson instance is used.

I would also prefer to use AllowAmbiguousTypes and `dataTypeFromBoundedEnum @Role`, but this isn't important.